pom.xml

```diff
@@ -121,9 +121,9 @@ flexible messaging model and an intuitive client API.</description>
     <testRealAWS>false</testRealAWS>
     <testRetryCount>1</testRetryCount>
 
-    <bookkeeper.version>4.7.0</bookkeeper.version>
+    <bookkeeper.version>4.8.0-SNAPSHOT</bookkeeper.version>
```

Comment:
We should try to avoid going back to use snapshot dependency here

Reply:
the plan is to use 4.8.0-SNAPSHOT once apache/bookkeeper#1441 is merged. and changed it to 4.7.1 once it is released. does that work for you?